lib/Mojo/Promise.pm

```diff
@@ -288,6 +310,27 @@ rejects. If the returned promise fulfills, it is fulfilled with the values from
 the fulfilled promises in the same order as the passed promises. This method can
 be useful for aggregating results of multiple promises.
 
+=head2 all_settled
+
+  my $new = Mojo::Promise->all_settled(@promises);
+  $new->then(sub ($results) {
+    for my $result (@$results) {
+      if ($result->{status} eq 'fulfilled') {
+        say "fullfiled with $result->{value}";
+      } else {
+        say "rejected with $result->{reason}";
+      }
+    }
+  });
```

Comment:
This example is very inconsistent.

Reply:
Update example. 

I'm not sure that I was right to add this two lines of comments
```perl
# fulfilled: 3
# rejected: foo
```

I didn't find any example in mojolicious docs, when we `say` something in code snippet and also print the output in the comment to that snippet. But in this case
1. we have a more complex structure of the result
2. this style inspired by the MDN example for Promise.allSettled.

Reply:
Please follow the same style the rest of the documentation uses.

Reply:
Changed the comment to `# check the status of promises`

Reply:
There seems to be a misunderstanding, this is not about comments. The examples as a whole are unlike everything else in the Mojolicious docs.

Reply:
What exactly differs this example from other in mojolicious docs? Mojo::UserAgent has [similar examples](https://metacpan.org/release/Mojolicious/source/lib/Mojo/UserAgent.pm#L742).
It works, it demonstrates usage and output format of `all_settled` method.

I'd love to change example, but don't understand in what way should i do that.